### DISCUSSION TITLE
Announcement of NFTs by Project X

### DISCUSSION THREAD

**Discussion by RPRX** [XTLS/Xray-core#3633] [Show and tell]
## Background

**自初始创立的近四年来，Project X 几乎是这里仅有的不收任何赞助、不收任何广告，也没有任何“产业”的项目组。** 因为我们创立 Project X 的初衷是捍卫人们自由获取与表达信息的基本人权，anti-censorship，do something good，而不是为了任何经济利益，即使我们已经投入了大量的时间精力、所创造的价值更是难以估量，我们也没有从中拿过一分钱，我想这已经足够表明我们的初心与态度。

然而没有人能一直坚持这样的~~纯倒贴~~，我本来打算以后以提供想法与建议、审核 PR 为主，~~让 contributors 当人工 copilot~~，但我发现即使是这样所耗费的时间精力也并不比以前自己直接写代码少，~~有时甚至一整天都泡在 GitHub 上讨论新协议~~，并且我还计划着更多的新东西、需要写更多的代码，使我不得不考虑以某种形式收取赞助，以弥补时间精力上的消耗，也能让我更加专注于这个项目。

## NFT

但是从几个隔壁的经验来看，“纯打赏”的效果并不好，而收广告的风险不一，~~开“产业”更是需要先润~~，所以我有一个大胆的想法：

**发行 NFT。**

具体而言，我计划以 Project X 所创造的项目、协议为名逐步打造 NFT，在该项目或协议的某个关键时间节点发行，~~为了发新 NFT 就只能多创新项目多写新协议~~，但每个数量只有 100 个，宁缺毋滥，其中 25 个会留作直接分发给相关贡献者们，以激励贡献，另外 75 个会公开售卖，你可以买来作永久收藏，也可以等升值后再卖出去，这比“纯打赏”强多了，~~逼格也更高~~。

注意：NFT 本身就有价值，持有 NFT 并不代表我们提供任何服务或你拥有了任何特权，~~比如不按模板开 issue 还是会被 close，入群问题答不对还是会被 ban~~，但你的确可以声称自己是 NFT 持有者，可以展示自己的集邮。

**目前第一个以 Project X 为名的 NFT 已在 OpenSea 上发行，这是第一个也是最特殊的 NFT，~~机不可失~~：**

https://opensea.io/assets/ethereum/0x5ee362866001613093361eb8569d59c4141b76d1/1

~~初始价格为 0.033 ETH，不会改，现在大概是 99 刀，老实说我不确定会多久售空~~，最近 REALITY 稳如老狗，所以没有那么多目光聚焦在这个领域，不像去年我回归前后那么火热，但这也是一个捡漏的好机会，希望大家用了这么久的 Xray 可以~~把握住机会~~支持一下，况且这不是“纯打赏”，价值会留在你手里，想变现时还可以卖出去，且极大概率比现在的初始价格高得多。

该 NFT 的赠予名单与数目，2024.8.5 更新第一批，2024.8.7、2024.8.10、2024.8.11 更新描述：

- 发频道后 v50 的好心人，一个（已转，他自己也买了一个）
- @yuhan6665，两个
- @Fangliding，一个（@mmmray 又送了一个给他，均已转）
- @mmmray，两个（已转一个，另一个送给了 @Fangliding ）
- @ll11l1lIllIl1lll，一个（已转）

剩下的会继续赠送给 Project X 以前、以后的核心贡献者们，~~建议他们使用 Tor 浏览器，虽然半天才能刷出一个能上 OpenSea 的 IP~~。
我对该 NFT 的未来充满信心，因为无论如何它都是这些代理软件所发行的第一个 NFT，~~我已经觉得 75 个放多了，希望不要卖完~~。

2024.8.7 更新：

这两天在思考 Project X NFT 的更多价值，比如有人说 0.033 ETH 较贵，日后会出价格低一些但数量多一些的 REALITY protocol NFT 以及价格更低但数量更多的 VLESS protocol NFT，提供更多选择，流动性也更强，并且出 REALITY protocol NFT 的时候给 Project X NFT 持有者送一个，以回馈他们在第一时间对 Project X 的支持。时机成熟时 Vision 和 SplitHTTP 也会出 NFT，会把原始收入转给它们的创造者 @yuhan6665 和 @mmmray ，~~虽然都是我给他们说要怎么写的~~。还有以后可以每年选一个对 Project X 贡献最多的非 Project X NFT 持有者送一个 Project X NFT，所以要多储备一些，正好我刚发现 OpenSea 的 listing 原来可以 cancel，打算卖到 20 个就暂停售卖，~~下一轮时预计只放 10 个并涨价到 199 刀，现在已经卖 13 个了且 ETH 的价格正触底，可以说是最划算的时候~~。

2024.8.8 更新：

由于近期 ETH 的价格大幅波动，我们已提前暂停 Project X NFT 的售卖，但已出售的 NFT 可以自由流通。目前我们手里还有 80 个去向未定的 NFT，打算以激励贡献（即赠予）为主要考虑。此外会不定期放出 5 个 NFT，但随着剩余数量的稀缺，价格会合理地逐次递增。

2024.8.9 更新：

新一轮放出了 5 个 Project X NFT，小幅上涨至 0.05 ETH。由于近期主要在处理 SplitHTTP 等 PR，尚未预备 Xray-core v1.9.0，可以趁机写完 REALITY 的文章，届时会一起放出 REALITY protocol NFT，价格低一些但数量多一些，并且会给届时的 Project X NFT 持有者送一个。

2024.8.10 更新：

昨天放出的 5 个 NFT 已被扫空，有点出乎意料，~~本来以为可以坚持几天到下一次发版~~，刚刚有人一下赞助了最后两个，非常感谢，~~但其实更建议只买一个，让持有者更多元~~。新一轮再次放出 5 个 Project X NFT，上涨至 0.1 ETH，希望可以坚持到下一次发版。

2024.8.16 更新：

为了预备发行 NFT 后 Xray-core 的首次发版，新一轮放出了 10 个 NFT，五个 0.15 ETH，五个 0.2 ETH。~~0.1 ETH 的 NFT 仅剩两个，想买来收藏的话需要抓紧时间入手~~。不久后会随着 REALITY 的文章放出 REALITY protocol NFT，会给届时的 Project X NFT 持有者送一个。

2024.10.18 更新：

昨天已售出第一个 0.15 ETH 的 Project X NFT，感谢支持！主要是在 0.1 ETH 横盘了近两个月，为了更充分体现“越早越便宜”的原则，今天把待售的 NFT 改为了 0.16、0.17 到 0.25 ETH 逐个递增。REALITY NFT 预计有一万个（送两个），0.01 ETH 一个，以便作为日常捐款途径。

2024.12.7 更新：

REALITY NFT 已上线一周，总量为一千个，它是一个相对平价的 NFT，如果你想捐 USDT/USDC 也可以，**当然最希望支持 Project X NFT：**

- **ETH/USDT/USDC: `0xDc3Fe44F0f25D13CACb1C4896CD0D321df3146Ee`**
- **Project X NFT: [Announcement of NFTs by Project X](https://github.com/XTLS/Xray-core/discussions/3633)**
- **REALITY NFT: [XHTTP: Beyond REALITY](https://github.com/XTLS/Xray-core/discussions/4113)**

**就像 Xray 开创过很多历史一样，发行 NFT 也是这个领域前无古人的操作。这些 NFT 非常有纪念意义，甚至可以说是有历史意义，远大于现在的初始价格，假以时日它们必将价值连城。最后再次感谢大家对 Project X 的支持。**

**Comment by iambabyninja** [comment]
IN RUSSIAN LANGUAGE: 

## Вступление

За почти четыре года с момента создания Project X он остается одной из немногих инициатив, которая не принимает спонсорскую помощь, не размещает рекламу и не имеет никаких коммерческих предприятий. Мы основали Project X с целью защищать основное право людей на свободный доступ к информации и свободное выражение мнений, бороться с цензурой, делать что-то хорошее, а не ради экономической выгоды. Несмотря на то, что мы вложили огромное количество времени и усилий и создали значительную ценность, мы не получили за это ни копейки. Я думаю, это достаточно ясно демонстрирует наши намерения и подход.

Однако никто не может постоянно работать в убыток. Я изначально планировал в будущем сосредоточиться на предоставлении идей и рекомендаций, а также на проверке PR, оставив разработку основным участникам проекта. Но я обнаружил, что даже такой подход требует не меньше времени и усилий, чем непосредственное написание кода. Иногда я целый день провожу на GitHub, обсуждая новые протоколы. У меня есть планы на множество новых проектов, требующих написания большего объема кода. Поэтому я вынужден рассмотреть возможность получения спонсорской поддержки, чтобы компенсировать затраты времени и усилий и более полно сосредоточиться на этом проекте.

## NFT

Однако, основываясь на опыте нескольких других проектов, чисто добровольные пожертвования неэффективны, реклама несет в себе разные риски, а создание бизнеса требует первоначальных вложений. Поэтому у меня возникла смелая идея: выпускать NFT.

Конкретно, я планирую выпускать NFT, связанные с проектами и протоколами, созданными в рамках Project X, в ключевые моменты развития этих проектов или протоколов. Чтобы выпускать новые NFT, придется создавать новые проекты и писать новые протоколы. Но каждую серию NFT я ограничу числом в 100 экземпляров, чтобы сохранить их уникальность. Из них 25 будут распределены среди участников проекта для стимулирования их вклада, а оставшиеся 75 будут выставлены на продажу. Вы можете купить их для постоянного хранения или продать позже по более высокой цене. Это намного лучше, чем просто принимать пожертвования, и выглядит гораздо престижнее.

Важно: сами NFT уже имеют ценность. Владение NFT не подразумевает получение каких-либо услуг или привилегий. Например, если вы создаете issue без соблюдения шаблона, она все равно будет закрыта, а неправильные ответы на вопросы при вступлении в группу все равно приведут к бану. Но вы действительно можете заявить, что являетесь владельцем NFT и показать свою коллекцию.

Первый NFT под названием Project X уже выпущен на OpenSea. Это первый и самый уникальный NFT, не упустите шанс:
[https://opensea.io/assets/ethereum/0x5ee362866001613093361eb8569d59c4141b76d1/1](https://opensea.io/assets/ethereum/0x5ee362866001613093361eb8569d59c4141b76d1/1)

Начальная цена составляет 0.033 ETH, что сейчас примерно 99 долларов. Честно говоря, я не уверен, как быстро они будут распроданы. В последнее время внимание к REALITY довольно стабильное, поэтому не так много интереса к этой области, как это было в прошлом году, когда я вернулся. Но это также отличная возможность для тех, кто давно использует Xray, поддержать нас. Тем более что это не просто пожертвование — ценность останется у вас, и вы сможете продать NFT в будущем, вероятно, за гораздо более высокую цену.

Позже я обновлю здесь список и количество подаренных NFT. Чтобы получить доступ к OpenSea через Tor-браузер, может потребоваться несколько попыток.

Как Xray уже создал много исторических прецедентов, так и выпуск NFT — это новаторский шаг в этой области. Эти NFT имеют большое значение для истории и будут стоить гораздо больше в будущем, чем их текущая начальная цена. Еще раз спасибо за поддержку Project X.

**Comment by Fangliding** [comment]
对吗
![IMG_20240803_225009_448](https://github.com/user-attachments/assets/1fdd0242-1ebb-41c4-877b-c3b001d50b65)

**Comment by Einsfier** [comment]
看来开源是真没办法当饭吃。

**Reply by Kissycat** [reply]
看项目吧，毕竟这个项目看起来就不太好认领..

**Reply by ghost** [reply]
> 看来开源是真没办法当饭吃。

一般可以，
~~但是這個項目的敵人，他可不是普通的競爭者只要消費者市場，他還要命，項目持有者需要消耗的精力相對更多~~

**Reply by bian2022** [reply]
> 看来开源是真没办法当饭吃。

大多数开源项目在找工作的时候会有一定的价值，但这个项目你在国内找工作的时候，敢承认是自己写的吗？

**Comment by kingwilliam** [comment]
可能我真的离开了cryptocurrency & NFT 太久了, 请问我在opensea.io payment时, 有以下讯息, 我应如何完成交易? thx

Error sending message to wallet: processing response error 
max priority fee per gas higher than max fee per gas

* 补充 : 我是有足够eth的
交易需要 0.033 + .0004 = 0.0334
我balance有0.0335eth

**Reply by RPRX** [reply]
感谢支持，再多一点点 ETH 应该就能完成交易了，~~我猜~~

**Reply by kingwilliam** [reply]
终於完成 真的要加多一点点 >.<

**Comment by PoneyClairDeLune** [comment]
Translation in English. Hopefully this would be as faithful as possible, I was struggling so much even with help from chatbots and someone who can speak Chinese...

## Background
**Since the founding of this project around four years ago, Project X is likely the only project that has never taken a single sponsorship, pushed a single advertisement, or fostered downstream industries first-hand.** We founded Project X on the basis of defending the basic human right of retrieving and distributing information, anti-censorship, and do something good, rather than generating financial profit for our own. Pouring an enormous amount of time and focus, creating immeasurable values in the meantime without taking a single penny, I guess this should be an evident enough proof that we've unwaveringly kept our original values and attitude.

In reality however, no one could keep on this trend of selfless devotion without any returns. Originally, I planned to only offer ideas and suggestions myself, auditing pull requests as a side measure, leaving the development work up to the contributors. But I quickly realized that this approach never shaved off the attention required than being directly involved myself, sometimes even getting myself discussing the new protocols on GitHub the entire day. I'm also drawing plans for lots of new features with code accompanying, forcing me to think of a way to gather sponsorship complementing my attention spent on this project, which could also help me focus on this project more in the long run.

## NFT
But judging from the experience of similar censorship circumvention projects... Tipping alone doesn't work well, advertisements bring an additional risk vector, and fostering a downstream industry requires me to wet myself first. Thus, a bold move from me:

**Releasing NFTs.**

To be more elaborate, I plan to mint NFTs based on the projects and protocols authored by Project X, and release them at the key moments of those corresponding branches; so if new NFTs are to be minted, new projects and protocols must be authored first. However, there would only be 100 for each one - better limit their quantity than letting them inundate everywhere. 25 will be reserved for distribution among related contributors to encourage contributions, with the remaining 75 sold publicly. Feel free to inflate your perpetual personal collection, or trade it once its value elevates; it's a better idea than tipping overall.

One thing to take notice: In possession of NFTs does not correlate that you're to be provided any service or to gain any privilege, as NFTs hold value themselves. Not following issue templates will still get those issues of yours closed, submitting wrong answers upon joining our group chat will still get you banned, however you can claim you own some of the NFTs, and display your prized collection.

**Here is the first NFT named after Project X, already released on OpenSea. This is the first and the most unique NFT ever released.**

https://opensea.io/assets/ethereum/0x5ee362866001613093361eb8569d59c4141b76d1/1

The initial price is 0.033 ETH, roughly 99 USD currently. It won't be changed in the future. I'm not sure when they will be sold out to be honest, as REALITY has been stable as hell lately, diverting focus from the field of censorship circumvention, unlike the heated focus it has around the time I returned last year. On the other hand though, this is the absolute gold chance of trading it for cheap, so I hope everyone can help support Xray after being its user for so long. It's not just tipping either, since you can trade it for cash whenever you want as you'll be holding its value, and it's highly probable that its price down the line will be way higher than its initial price.

List of the NFTs gifted on the first round, updated on 5th August:
- One to the one who (WTF?) after the creation of the channel (he bought one as well)
- Two to @yuhan6665
- One to @Fangliding
- Two to @mmmray
- One to @ll11l1lIllIl1lll

The remaining ones will be gifted to the past and future contributors continuously to the core contributors. I advise them to use the Tor Browser, though you'll only roll a single IP not blocked by OpenSea after a good long while. I'm having full confidence in the future of this NFT, since this would be the first NFT ever released by the encrypted proxies. 75 should be more than enough, wish it wouldn't be sold out.

**Just like Xray making history for countless times before, the release of this NFT will be the new shiny chapter for the anti-censorship community. These NFTs are highly commemorative, to the point of being a significant segment of the history, valuing them several magnitudes higher than what their initial price could ever be. With enough time, they'll prove to be invaluable. We appreciate everyone for their unwavered support on Project X.**

**Comment by hax0r31337** [comment]
May I get a NFT please, I have contributed WireGuard Inbound (#2477).
Here's my ETH address 0x4aA0930fB031cc81140231F59AE79186F9Ad07Ba

**Reply by RPRX** [reply]
首先感谢你的贡献，我看了一下，如果送的话我们还要给 WireGuard Outbound 的贡献者送一个，它的使用率更高（虽然和其它协议比还是差远了。。），这样很快就不够送了，**但是如果你能 PR 一个 Windows 的 TUN inbound，我直接送两个**，我们三年前就有过测试 https://github.com/XTLS/Xray-tun ，不过好像需要 wintun.dll，Clash 它们的实现方式好像不一样，最好不要外部 dll 吧

**Reply by RPRX** [reply]
你在研究 Tun inbound 吗，~~如果没有的话我们可以公开发榜等有缘人揭榜~~

**Comment by Candinya** [comment]
支持了，感谢一直以来的持续开发和维护，为众人抱薪。愿此般绵薄之力能给您和大家的未来带来更多帮助。

**Reply by Natsuki-Kaede** [reply]
捉

**Reply by KipJayChou** [reply]
捕捉糖喵大佬

**Comment by PMExtra** [comment]
玩币圈需要炒热度和知名度的，小众人群圈地自萌还能升值就有点见鬼了。
建议整点初始价值较低的东西，然后给所有contributor来个空投[doge]

**Reply by RPRX** [reply]
这件事的本质是捐款，但我想给大家一个纪念品，所以形式是 NFT，并不是币圈那些炒作项目。由于 Project X 是这些代理软件所发行的第一个 NFT，有历史意义，我们还在拿它奖励给贡献者，有实际意义，未来可能还会有实际作用，在数量稀缺的前提下，我觉得未来升值是不可避免的，光是官方售价都会逐次递增。这些代理软件的开发是小众，但用户群体可并不小众，知名度不会是问题。

**Comment by Scott169** [comment]
心酸，开发平台的人赚不到钱，赚钱的反倒是用来开机场的

**Comment by shinnku-nikaidou** [comment]
买了一个, 支持一下, 祝项目能活到gfw倒塌那一天

**Reply by lxsq** [reply]
等GFW撤了这项目都死不了，毕竟电信APP都在用🤣。

**Reply by Alceatraz** [reply]
> 等GFW撤了这项目都死不了，毕竟电信APP都在用🤣。

啊？？

**Reply by fuokgfw** [reply]
> > 等GFW撤了这项目都死不了，毕竟电信APP都在用🤣。
> 
> 啊？？

有根据么

**Reply by 8MiYile** [reply]
@fuokgfw https://x.com/nek0hasekai/status/1717786966253858978
https://www.v2ex.com/t/986532

**Comment by ssmetall** [comment]
I want to donate but I don't know what to do? Is it enough to just transfer this ethereum to buy nft? Isn't it better to put a crypto wallet so we can send donations to it? Like Tether and Tron

**Reply by RPRX** [reply]
使用 MetaMask 登录 OpenSea 就可以用 ETH 购买 NFT 了，目前只想通过 NFT 的形式收捐款

**Comment by gydchenxiao** [comment]
我还是希望Project X社区日后可以出价格低一些但数量多一些的 REALITY protocol NFT 以及价格更低但数量更多的 VLESS protocol NFT，可以提供更多选择，作为一个学生，还是愿意出0.01的eth，20多u直接购买一个nft，以表示我对xray的敬意和支持🙏。

**Reply by RPRX** [reply]
感谢你的支持，~~应该快了~~

**Comment by dalaohuuu** [comment]
刚玩墙半年，我i有条件也有意愿收藏NFT支持项目，可是在墙内多年，对加密货币如何交易毫无了解，还没把rmb弄到opensea那个eth wallet里，我是不是太笨了

**Reply by t4nature** [reply]
如果你人在墙内，建议你在学会用xmr切断追踪之前谨慎操作，不然被党卫军顺着nft找交易所跟他们要你的实名认证信息就糟糕了

**Reply by Alceatraz** [reply]
> 如果你人在墙内，建议你在学会用xmr切断追踪之前谨慎操作，不然被党卫军顺着nft找交易所跟他们要你的实名认证信息就糟糕了

那如果我就是想打赏一下怎么搞？

**Reply by t4nature** [reply]
https://t.me/projectXtls/277

理论上你不止可以发eth给他，USDC/USDT只要发给这个地址他都能收到，甚至不同链都行(需要和eth链机制相似，比如polygon和avax)

**Comment by t4nature** [comment]
可以考虑学一下[友站](https://v2.hysteria.network/zh/docs/Donation/#__tabbed_1_1)的捐赠支持模式，

Nft具有收藏和展示价值也是很不错的事情，尤其是对于一些名人(网红)来说他们不介意花钱为自己买个勋章，而这对于宣传项目(捐款页面)也有好处。这方面我能提的建议就是考虑交易费用更低的链(Solana、polygon)，以及早日发布单价更低的nft。

我一直认为做好事不要羞于谈钱。相反，有足够的钱才能把伟大的事业**可持续**地**以更高效率**地推动下去

**Reply by liaoran** [reply]
极对，理想主义最终都会被现实主义击败，没钱吃饭还怎么整伟大的事业，何况还要交电费、水费。

**Comment by hanjia115** [comment]
看不太懂？，这个资助途径是加密货币吗？你可以发支付宝或者微信收款码我资助一些钱，毕竟用了这么多年了饮水思源，如果出于保密PayPal也可以。

**Reply by bian2022** [reply]
> 看不太懂？，这个资助途径是加密货币吗？你可以发支付宝或者微信收款码我资助一些钱，毕竟用了这么多年了饮水思源，如果出于保密PayPal也可以。


……
用国内的付款方式，他们怕出事

**Reply by gydchenxiao** [reply]
> 看不太懂？，这个资助途径是加密货币吗？你可以发支付宝或者微信收款码我资助一些钱，毕竟用了这么多年了饮水思源，如果出于保密PayPal也可以。

对，eth

**Reply by alexallen1** [reply]
支付宝或者微信肯定不合适吧 拉清单抓的死死的

**Reply by liaoran** [reply]
我感觉，可以多准备几个支付宝或者微信号，作者不说，谁知道是干啥用的，问就是整知识付费，加密货币这东西整不明白啊，我还是过完年找到工作再研究下，现在有心也无力，难顶

**Comment by hanjia115** [comment]
那些靠这个挣钱的机场主，代理商们应该饮水思源拿出一些钱来资助开发者，毕竟你们赚的盆满钵满！~！

**Comment by hugetiny** [comment]
我觉得吧，作者别靠NFT这种邪道了。现实中版权用来打官司，虚拟世界中数字版权有什么实际意义吗？
比特币实际的价值在于洗钱，以前的人靠网络之外的方式可能是艺术品，或者古董转卖什么的，远没有比特币方便。
其实还是建议作者先润，再考虑后面的，一共活那么多年，那么憋屈干什么。

**Comment by alexallen1** [comment]
放几个币地址 说不定世界各地都有人打赏

**Reply by Sheva-ws** [reply]
赞同

**Comment by liaoran** [comment]
Oh, no, 99美元接近1000元，躺平三年，没有积蓄还负债的我摸了摸空空的钱包，看了看空空的银行卡，陷入了沉思，希望过完年没本事的我能找到工作再买吧。

话说，以后的新版本要花钱才能用了吗？

**Reply by ACnoway** [reply]
xray应该是永远免费的，这只是一个打赏渠道

**Comment by onediram** [comment]
😁😁😁😁😁

сб, 18 янв. 2025 г., 11:09 AM liaoran ***@***.***>:
